lib/core.ps1

```diff
@@ -1298,8 +1298,14 @@ function Out-UTF8File {
 Optimize-SecurityProtocol
 
 # Load Scoop config
+$configFilename = 'config.json'
```

Comment:
Why separate variable for filename?

Reply:
It's just a _draft_ for now. there are problems with this implementation, it'll try to access the wrong location when Scoop is cloned to an unpredictable folder for use/test for instance.

Reply:
Oh okay